rec_action/response_type/recommend_resp.py

```diff
@@ -7,6 +7,10 @@ class RecommendResponse(Response):
 
     :param domain: domain of the recommendation (e.g. restaurants)
     """
+
+    _current_recommended_items: list
```

Comment:
use list[RecommendedItem] if possible. also add return type to get_current_recommended_items function